optimum/exporters/executorch/quantization.py

```diff
@@ -101,6 +102,13 @@ def build_linear_config(quant_config_key: str, granularity: str, packing_format:
                     weight_dtype=torch.int8,
                     weight_granularity=PerAxis(0),
                 )
+            if quant_config_key == "fpa4w":
+                # Need to import to load the ops
+                import torchao.experimental.ops.mps  # noqa: F401
```

Comment:
nit should we import this in torchao.experimental.quant_api so that `from torchao.experimental.quant_api import UIntxWeightOnlyConfig` can satisfy the import requirement?

Reply:
that's also why I import `torchao.experimental.ops.mps` only if `quant_config_key == "fpa4w"`